.github/workflows/deploy-pages.yml

```
@@ -0,0 +1,87 @@
+
+      - run: corepack enable
+
+      - env:
+          HUSKY: 0
+        run: pnpm install --frozen-lockfile --prefer-offline
+
+      - env:
+          BASE_PATH: /dantalion/
+        run: pnpm --filter @kurone-kito/dantalion-web-demo-web run build
+
+      - if: ${{ secrets.DANTALION_GH_PAGES_DEPLOY_KEY == '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)